srcpkgs/gnome-flashback/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'gnome-flashback'
+pkgname=gnome-flashback
+version=3.30.0
+revision=1
+_submodule_commit=ce8e4880ce31e275c40825c4ed756c791107f810
+build_style=gnu-configure
+configure_args="--disable-schemas-compile --enable-compile-warnings=minimum
+ --enable-gtk-doc"
+hostmakedepends="automake autogen autoconf-archive gettext-devel intltool
+ libtool gobject-introspection gtk-doc intltool libxslt pkg-config python3
+ sassc git"
+makedepends="caribou-devel folks-devel
+ gjs-devel gnome-bluetooth-devel gnome-control-center-devel gnome-desktop-devel
+ gnome-menus-devel ibus-devel libcanberra-devel libcroco-devel mutter-devel
+ network-manager-applet-devel polkit-devel pulseaudio-devel
+ startup-notification-devel telepathy-logger-devel upower-devel"
+depends="gnome-bluetooth gnome-session gnome-settings-daemon
+ gnome-themes-standard"
+short_desc="GNOME 3 session with similar user experience to the GNOME 2.x series"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://wiki.gnome.org/Projects/GnomeFlashback"
+distfiles="https://gitlab.gnome.org/GNOME/gnome-flashback/-/archive/${version}/${pkgname}-${version}.tar.gz
+ https://gitlab.gnome.org/GNOME/libgnome-volume-control/-/archive/ce8e4880ce31e275c40825c4ed756c791107f810.tar.gz"
```

Comment:
why don't you use `${_submodule_commit}.tar.gz` ?

Reply:
sorry missed that. fix it asap.

Reply:
please do not add any more commits, just use `git commit --amend`

Reply:
Alright. I wanted to squash it into one commit anyway. But first I want to have clean template...

Reply:
Why don't you use use the archives from https://download.gnome.org/sources/gnome-flashback? Would spare you from having to download the submodules and from running autorgen.sh

Reply:
@Cogitri Thanks for this hint. Because I didn't know that GNOME provides this. Of course this is much better. Right now I am not able to do any changes because I am on the road but I will change this asap.

Reply:
I suppose it doesn't matter, you can be the maintainer too, but if you want me to be the maintainer  for gnome-flashback that'd be fine by me too

Reply:
So, I think about it and report soon. Just want the best for Voidlinux. :)